cms/static/js/views/utils/view_utils.js

```diff
@@ -175,7 +175,7 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
 
         var keywordValidator = (function () {
             var regexp = /%%[^%\s]+%%/g;
-            var keywordsSupported = ['%%USER_ID%%', '%%USER_FULLNAME%%', '%%COURSE_DISPLAY_NAME%%', '%%COURSE_END_DATE%%'];
+            var keywordsSupported = ['%%USER_ID%%', '%%USER_FULLNAME%%', '%%COURSE_DISPLAY_NAME%%', '%%COURSE_START_DATE%%', '%%COURSE_END_DATE%%'];
```

Comment:
Given that list line continues to grow, can we split it up and put each item on a separate line?